### PR TITLE
[BottomNavigationBar] Typography Implementation.

### PR DESCRIPTION
**Pull request by mohammadcazig**
BottomNavigationBar Typography Themer
Pivotal: https://www.pivotaltracker.com/story/show/156561773
Implementation
Pod target,
unit tests,
example update.

screen shot 2018-04-10 at 11 55 52 am

### CODE REVIEW

**Review by jverkoey — APPROVED** [review]

File: components/BottomNavigation/tests/unit/BottomNavigationTypographyThemerTests.m [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+@implementation BottomNavigationTypographyThemerTests
+
+- (void)testTypogrpahyThemer {
+  MDCTypographyScheme *typographyScheme = [[MDCTypographyScheme alloc] init];
+  MDCBottomNavigationBar *bottomBar = [[MDCBottomNavigationBar alloc] initWithFrame:CGRectZero];
+  UITabBarItem *item1 = [[UITabBarItem alloc] initWithTitle:@"1" image:nil tag:0];
+  UITabBarItem *item2 = [[UITabBarItem alloc] initWithTitle:@"2" image:nil tag:0];
+  bottomBar.itemTitleFont = [UIFont systemFontOfSize:31];
+  bottomBar.items = @[item1, item2];
```
Comment (jverkoey) [inline components/BottomNavigation/tests/unit/BottomNavigationTypographyThemerTests.m:33]: `bottomBar.items` isn't being used in this test. Can we make use of it somehow?

Comment (mohammadcazig) [reply components/BottomNavigation/tests/unit/BottomNavigationTypographyThemerTests.m:33]: Oops I saw this after I merged :(

Comment (romoore) [reply components/BottomNavigation/tests/unit/BottomNavigationTypographyThemerTests.m:33]: I think they can be removed unless you want to inspect the private views.

File: components/BottomNavigation/tests/unit/BottomNavigationTypographyThemerTests.m [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#import "MDCBottomNavigationBarTypographyThemer.h"
+#import "MaterialBottomNavigation.h"
+
+@interface BottomNavigationTypographyThemerTests : XCTestCase
+
+@end
+
+@implementation BottomNavigationTypographyThemerTests
+
+- (void)testTypogrpahyThemer {
```
Comment (romoore) [inline components/BottomNavigation/tests/unit/BottomNavigationTypographyThemerTests.m:27]: Nit: Spelling of method name

File: components/BottomNavigation/src/TypographyThemer/MDCBottomNavigationBarTypographyThemer.h [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ Applies a Typography scheme to an MDCBottomNavigationBar.
+ */
+@interface MDCBottomNavigationBarTypographyThemer : NSObject
+
+/**
+ Applies a typography scheme to theme an MDCBottomNavigationBar.
+
+ @param typographyScheme The typography scheme to apply to an MDCBottomNavigationBar.
```
Comment (romoore) [inline components/BottomNavigation/src/TypographyThemer/MDCBottomNavigationBarTypographyThemer.h:31]: Nit: Please reference `@c bottomNavigationBar` as the target of the scheme.

File: components/BottomNavigation/src/TypographyThemer/MDCBottomNavigationBarTypographyThemer.h [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ @param bottomNavigationBar An MDCBottomNavigationBar instance to apply a typography scheme to.
```
Comment (romoore) [inline components/BottomNavigation/src/TypographyThemer/MDCBottomNavigationBarTypographyThemer.h:32]: Nit/suggestion: The MDCBottomNavigationBar instance to theme.

**Pull request merged**